7.3.Частые_сценарии/7.3.14.py

```diff
@@ -0,0 +1,11 @@
+# Only even numbers 🌶️
+
+Flag = True
+for i in range(10):
+    n = int(input())
+    if n % 2 != 0:
+        Flag = False
+if Flag:
+    print("YES")
+else:
+    print("NO")
```

Comment:
Можно так, а можно чуть успростить:
```suggestion
flag = "YES"

for _ in range(10):
    n = int(input())
    if n % 2 != 0:
        flag = "NO"

print(flag)
```